### DISCUSSION TITLE
Dead cells

### DISCUSSION THREAD

**Discussion by masterpiece67** [skyline-emu/skyline#1318] [Game Requests]
![Screenshot_2022-05-15-20-30-57-292_skyline emu](https://user-images.githubusercontent.com/92961132/168473027-4d040626-c6b3-4121-8a22-410864562e9f.jpg)

Dead cells can finally booting, but stuck on the loading screen and I've tried to off and on docked mode, still stuck. I hope the dev can fix it.

**Comment by Mhsamk** [comment]
Bro you can play it without an emulator

**Reply by meizano** [reply]
There's an android version of Dead Cells in Play Store

**Comment by Crytonics** [comment]
Game is on the list: https://github.com/skyline-emu/skyline-games-list/issues/369

**Comment by ghost** [comment]
buy the game legit instead of pirating it.
